config/locales/blacklight.it.yml

```diff
@@ -0,0 +1,226 @@
+        failure:  'Si è verificato un errore nel salvare i preferiti.'
+      remove:
+        button: 'Togli la pagina dai preferiti'
+        success: 'La pagina è stata tolta dai preferiti.'
+        failure: 'Si è verificato un errore nel togliere la pagina dai preferiti.'
+        action_confirm: 'Vuoi togliere questa pagina dai preferiti?'
+      clear:
+        action_title: 'Cancella tutti i preferiti'
+        action_confirm: 'Vuoi cancellare tutti i preferiti?'
+        success: 'Tutti i preferiti sono stati cancellati.'
+        failure: 'Si è verificato un errore nel cancellare i preferiti.'
+      need_login: 'Effettuare il login per gestire e visualizzare i preferiti.'
+      list_title: 'Preferiti'
+      delete: 'Togli'
+
+    saved_searches:
+      add:
+        success: 'Ricerca salvata.'
+        failure: 'Si è verificato un errore nel salvare la ricerca.'
+      remove:
+        success: 'La ricerca è stata tolta dalla lista.'
+        failure: 'Si è verificato un errore nel togliere la ricerca dalla lista.'
+      clear:
+        action_title: 'Cancella tutte le ricerche salvate'
+        action_confirm: 'Vuoi cancellare tutte le ricerche salvate?'
+        success: 'Tutte le ricerche salvate sono state cancellate.'
+        failure: 'Si è verificato un errore nel cancellare le ricerche salvate.'
+      title: 'Ricerche salvate'
+      need_login: 'Effettuare il login per gestire e visualizzare le ricerche salvate.'
+      no_searches: 'Non è stata salvata nessuna ricerca.'
+      list_title: 'Ricerche salvate'
+      delete: 'cancella'
+
+    search_history:
+      clear:
+        action_title: 'Cancella le ricerche effettuate'
+        action_confirm: 'Vuoi cancellare le ricerche effettuate?'
+        success: 'Le ricerche effettuate sono state cancellate.'
+        failure: 'Si è verificato un errore nel cancellare le ricerche effettuate.'
+      title: 'Ricerche effettuate'
+      no_history: 'Non è stata effettuata nessuna ricerca'
+      recent: 'Ricerche effettuate'
+      forget: 'dimentica'
+      save: 'salva'
+
+    tools:
+      title: 'Strumenti'
+      citation: 'Cita'
+      endnote: 'Esporta in EndNote'
+      refworks: 'Esporta in Refworks'
+      email: 'Invia per e-mail'
+      sms: 'Invia per SMS'
+      clear: 'Cancella'
+
+    citation:
+      mla: 'MLA'
+      apa: 'APA'
+      chicago: 'Chicago'
+
+    email:
+      form:
+        title: 'Invia per e-mail'
+        to: 'E-mail:'
+        message: 'Messaggio:'
+        submit: 'Invia'
+      text:
+        title: 'Titolo: %{value}'
+        author: 'Autore: %{value}'
+        format: 'Formato: %{value}'
+        language: 'Lingua: %{value}'
+        subject:
+          one: 'Numero di scheda: %{title}'
+          other: 'Numeri di scheda'
+        url: 'URL: %{url}'
+        message: 'Messaggio: %{message}'
+
+      success: "Messaggio inviato"
+
+      errors:
+        to:
+          invalid: 'L'indirizzo e-mail inserito non è valido'
```

Comment:
You should escape the single quote here (and line 123). Per the [YAML spec](http://yaml.org/spec/current.html#id2534365), I believe you should use `''` (two single quotes) here, or change the surrounding single quotes to double quotes.